content/source/docs/cloud/integrations/splunk/index.html.md

```diff
@@ -42,6 +42,7 @@ The current release of the Terraform Cloud for Splunk app supports the following
 ## Installation & Configuration
 
 * Install the [Terraform Cloud for Splunk app via Splunkbase](https://splunkbase.splunk.com/app/5141/)
+  * Splunk Cloud users should consult the latest instructions on how to [use IDM with cloud-based add-ons](https://docs.splunk.com/Documentation/SplunkCloud/8.0.2007/Admin/IntroGDI#Use_IDM_with_cloud-based_add-ons) within the Splunk documentation.
```

Comment:
```suggestion
    * Splunk Cloud users should consult the latest instructions on how to [use IDM with cloud-based add-ons](https://docs.splunk.com/Documentation/SplunkCloud/8.0.2007/Admin/IntroGDI#Use_IDM_with_cloud-based_add-ons) within the Splunk documentation.
```

If that's meant to be a nested list item under the first one, it should have exactly four spaces.